sdks/python/apache_beam/runners/runner.py

```diff
@@ -116,8 +116,21 @@ class PipelineRunner(object):
   materialized values in order to reduce footprint.
   """
 
-  def run(self, pipeline):
-    """Execute the entire pipeline or the sub-DAG reachable from a node."""
+  def run(self, transform, options=None):
```

Comment:
Test cases? You can use runner_test.py in https://github.com/apache/beam/pull/4223/commits/9d87830c9ac311b8069a7e4a4f6470ba2e3223a6
and change runner.run() calls to runner.run_pipeline() if a pipeline is provided as input.

Reply:
This is tested at test_run_api. 

We don't want to accept runner.run(pipeline) as part of the public api.